src/Controller/Api/MailArchiveController.php

```diff
@@ -104,13 +105,13 @@ public function download(Request $request, Context $context): JsonResponse
 
         $content = $this->getContent($mailArchive->getEmlPath());
 
-        if (empty($content)) {
+        if ($content === '' || $content === '0' || $content === false) {
```

Comment:
rector seems to transform the empty call into something kind of non-sense, doesn‘t it? f.e. „0“. when we make the check explicitly it should be fit to the input.

this relates to a few changes.

Reply:
we should disable that rule right? I don't see the benefit of doing kinda the same what empty is doing just manually